check-ostree.yaml

```diff
@@ -213,6 +213,27 @@
       when:
         - "'/dev/mapper/rootvg-rootlv' in result_sysroot_source.stdout"
         - fdo_credential == "true"
+        - os_name == "redhat"
+
+    # on fedora, it grows to 18.49G
+    - name: "pv size should bigger than 18G"
+      block:
+        - assert:
+            that:
+              - "'18' in result_pv_size.stdout"
+            fail_msg: "pv size is not bigger than 18G"
+            success_msg: "pv size is bigger than 18G"
+      always:
+        - set_fact:
+            total_counter: "{{ total_counter | int + 1 }}"
+      rescue:
+        - name: failed count + 1
+          set_fact:
+            failed_counter: "{{ failed_counter | int + 1 }}"
+      when:
+        - "'/dev/mapper/rootvg-rootlv' in result_sysroot_source.stdout"
+        - fdo_credential == "true"
+        - os_name == "fedora"
```

Comment:
could you please use `ansible_facts['distribution']` for different distros.

Reply:
Fixed.